compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/methodgen/MainMethodGen.java

```diff
@@ -156,15 +168,36 @@ private void generateMethodCall(String initClass, MethodVisitor mv,
         genReturn(mv, indexMap, futureVar);
     }
 
+    private void generateGetFilePath(ClassWriter cw) {
```

Comment:
Can we move this to runtime package and just generate static call instead of generating the full method.